### PR TITLE
#110 Fix exit command

### PR DESCRIPTION
**Pull request by Hiroaki-K4**
- Add error handling for numbers larger or smaller than long.
- Add error handling non-numeric exit arguments.

### CODE REVIEW

File: srcs/builtins/exit.c [hunk 1]
```diff
@@ -6,18 +6,114 @@
...
+			continue ;
+		}
+		if (ft_isdigit(str[i]) == 0)
+			return (FALSE);
+		i++;
+	}
+	return (TRUE);
+}
+
+static	long	calc(const char *str, long cutoff, long cutlim, int is_minus)
```
Comment (cheedah) [inline srcs/builtins/exit.c]: return type: long -> int

File: srcs/builtins/exit.c [hunk 2]
```diff
@@ -6,18 +6,114 @@
...
+				return (TRUE);
+			else
+				res = 10 * res + (*str - '0');
+		}
+		str++;
+	}
+	return (FALSE);
+}
+
+static	long	check_over_long(const char *str, int is_minus)
```
Comment (cheedah) [inline srcs/builtins/exit.c]: return type: long -> int

File: srcs/builtins/exit.c [hunk 3]
```diff
@@ -6,18 +6,114 @@
...
+
 int	ft_exit(char **argv, t_envs **envs)
 {
 	(void)envs;
 	ft_putendl_fd("exit", 1);
 	if (argv[1] == NULL)
 		exit(EXIT_SUCCESS);
+	else if (is_all_digit(argv[1]) == FALSE || is_over_long(argv[1]))
+	{
+		ft_putstr_fd("minishell: exit: ", 1);
```
Comment (cheedah) [inline srcs/builtins/exit.c]: use stderr fd

**Review by cheedah — APPROVED** [review]

File: srcs/builtins/exit.c (hunk 1 above)
Comment (Hiroaki-K4) [reply srcs/builtins/exit.c]: Fixed.

File: srcs/builtins/exit.c (hunk 3 above)
Comment (Hiroaki-K4) [reply srcs/builtins/exit.c]: Fixed.

File: srcs/builtins/exit.c (hunk 2 above)
Comment (Hiroaki-K4) [reply srcs/builtins/exit.c]: Fixed.

**Pull request merged**